File: Android/TwinRinksAdultHockey/src/com/gigaStorm/twinRinks/Team.java
package com.gigaStorm.twinRinks;

// Wrapper class for an object representing a team
public class Team {

    private String league;
    private String teamName;

    public Team(String l,String n) {
	league = l;
	teamName = n;
    }

    public Team(String backing) {
	String[] data = backing.split(";");
	league = data[0];
	teamName = data[1];
    }

    public String getTeamKey() {
	return league + ";" + teamName;
    }

    public String getLeague() {
	return league;
    }

    public String getTeamName() {
	return teamName;
    }

    public String toString() {
	return league + "-" + teamName;
    }
}

File: iOS/TwinRinksAdultHockey/TwinRinksAdultHockey/Team.m
//
//  Team.m
//  TwinRinksAdultHockey
//
//  Created by Andrew on 2/28/13.
//  Copyright (c) 2013 GigaStorm. All rights reserved.
//

#import "Team.h"

@implementation Team

@synthesize teamName,league;

-(id)initWithTeam:teamNameP andLeague:leagueP {
    self = [super init];
    
    self.teamName = teamNameP;
    self.league = leagueP;
    
    return self;
}

-(id) init {
    self = [super init];
    return self;
}

-(NSString *) toString {
    NSString *toReturn = [NSString stringWithFormat:@"%@ - %@",self.teamName,self.league];
    return toReturn;
}

-(NSString *) getTeamKey {
    NSString *toReturn = [NSString stringWithFormat:@"%@;%@",self.teamName,self.league];
    return toReturn;
}

@end
